api/rs/slint/lib.rs

```diff
@@ -177,7 +177,7 @@ The follow table summarizes the entire mapping:
 | `bool` | `bool` | |
 | `brush` | [`Brush`] | |
 | `color` | [`Color`] | |
-| `component-factory` | [`ComponentFactory`] | A factory for components that can be added at runtime. |
+| `component-factory` | [`ComponentFactory`] | A factory for components that can be added at runtime. (only available with `support_component_container` set to `true` in `CompilerConfiguration`)|
```

Comment:
Small comment on the side: I think this should be removed. The API you're adding a field to is not public. The public `CompilerConfiguration` is not a struct with public fields. See [here](https://slint.dev/releases/1.1.1/docs/rust/slint_build/struct.CompilerConfiguration).